docs/_plugins/angular-leaflet-directive.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Angular Leaflet directive
+category: frameworks-&-build-systems
```

Comment:
I would rather not have `&` in the category names. I don't trust all YAML parsers will play nicely with that.

Please modify your script (at https://github.com/Falke-Design/Leaflet/commit/f00794cc5d36c443afd004afe9756aeac97a674d#diff-41c60cd16ab76d71244c42809a6f886d662c8c676b3de8f2fe1e5f61ccafee6eR128 ) to provide category names without `&`s.